cli/installer/win/cmd.go

```diff
@@ -0,0 +1,44 @@
+package win
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+func ExecCmd(cmd string, suppressOutput bool) error {
```

Comment:
added a `suppressOutput` option here so the CLI doesn't show "Refreshing environment variables" every time we run `refreshEnvVariables`. There's no option to make this command quiet. So I added this here

Reply:
You can "mute" the cmd: https://stackoverflow.com/a/6461021